manifold/src/boolean3.cpp

```diff
@@ -14,6 +14,7 @@
 
 #include "boolean3.h"
 #include <limits>
+#include <thrust/execution_policy.h>
```

Comment:
Not that it really matters, but I might just add this include to vec_dh.h, instead of every file that uses it.

Reply:
yes, I just rushed for a PR so I did not clean it up properly, will fix